### PR TITLE
Strips the Disabler from Human/Cyborg use

### PR DESCRIPTION
**Pull request by Ergovisavi**
The disabler was originally implemented by me in order to serve as a proof of concept of a "no more one shots" address to the stun system, while also attempting to marry it into the already existing health system. While I did eventually open up a PR for such a thing, a different PR was chosen.

Tasers and Disablers are diametrically opposed gameplay concepts and cannot properly work in the same space. In order to compete with tasers, disablers have to be very high damage lasers (They're at 36 currently, as opposed to lasers, which deal 20), go through glass, have laser range, etc. What's more, they're used by the same people (Security) for the same purpose (Stunning someone to end a combat encounter) and in some cases are basically just used AFTER a fight to double-super-absolutely make sure someone stays down nonlethally.

There's also a design tax, because tasers and disablers work off slightly different systems, things like the adrenaline implant needs to account for both possibilities, changelings ran into this kind of crap, etc. They just don't need to both exist and one of them needs to go.

TL;DR, we don't need both of these, one of them needs to go. They are a quarter implemented system, implemented in a haphazard way.

If you're curious, the original intention of the disabler was to:
* Allow you to fight back if someone draws a stun weapon on you
* Marry the projectile stun system and the health system into one function, so your health still very much matters at all times instead of the taser's 'lol whatever' approach
* Allow more, less effective, ammo to be carried in a single weapon so that after a rookie sec officer won't instantly blow his load and have to rely on our garbage melee click system
* Curb instant stuns from down the hallway (the range function on projectiles was created for this), but then disablers were turned into long range, window piercing lasers

:cl:
del: Stripped the disabler from standard Human/Borg use, we're back to just tasers and laser
/:cl:

### CODE REVIEW

**Comment by FantasticFwoosh** [comment]
Freeze is going to stop you right there for balance changes, though there are still chemically applied stamina depleting alternatives via darts.

**Comment by Ergovisavi** [comment]
That's entirely fine, this is to strip out the main source of design space competition that they exist in.

**Comment by lzimann** [comment]
Freeze

**Comment by drfino** [comment]
Tasers are high risk, high reward. Disablers are low risk, low reward. Why can't they both exist? They both have a purpose. Just because they are two similar methods doesn't mean one "needs to go." That's dangerous logic there.

**Comment by msgerbs** [comment]
Tasers have a huge flaw in that they can't shoot somebody moving away from you unless they are very close, because they hit the tile they are shot at. A disabler will keep going and hit a fleeing target. If anything, remove tasers for being useless garbage.

**Comment by FantasticFwoosh** [comment]
How about we remove both and instead rework tasers to be more like real life with the little cable shooting out thing that attaches to people 1,2 or three tiles away but continually shocks them until it runs out of charge?

**Comment by msgerbs** [comment]
Do you want sec to just carry lethals? Because that's how you get sec to just carry lethals.

**Comment by PKPenguin321** [comment]
>Tasers are high risk, high reward. Disablers are low risk, low reward.

no, disablers are low risk, very high reward. slowdown is disgustingly thick and the stun is longer and easier to stack.

**Comment by Ergovisavi** [comment]
>no, disablers are low risk, very high reward

Sort of. Depending on the context, such as fighting against someone else who has a taser, disablers are highly risky to use instead of your taser, because you'll get dropped in one hit, but you need to hit them with three. Their reward isn't instant, unlike tasers. Once you actually get someone stunned, yes, they're effectively infinite reward.

**Comment by drfino** [comment]
@PKPenguin321 the high reward for taser is instant stun. the low reward for disabler is stun after three hits

**Comment by DoctoreMad** [comment]
>They both have a purpose. Just because they are two similar methods doesn't mean one "needs to go."

Are you ignoring the part where he writes out how the clunky code causes problems with stun resistance or?

**Comment by CPTANT** [comment]
We discussed this in https://tgstation13.org/phpBB/viewtopic.php?f=9&t=12130

Stamina should be reworked with reduced slowdown and by removing stamina damage after your stun ends.

**Comment by Steelpoint** [comment]
Disablers are good for two reasons. You get more shots to fire and a single hit can incur sufficent slow down to make follow up hits easier.

**Comment by Ergovisavi** [comment]
>Reduced Slowdown

It's health slowdown. You're asking for health slowdown to be tweaked, which I am for, but is a larger scope issue.

>Removing stamina damage after your stun ends

Okay so not only does this not solve being tased after falling on the ground, so chainstunning is still a thing, you now need to account for stuns placed after the player is stunned, you also introduce gameplay confusion by creating situations where you're better off hitting them with a taser at the end instead of continuing to fire your disabler because that way they keep the slowdown if they get up, people are still going to shoot you with their disabler after you're cuffed "just in case", etc.

It doesn't work and them existing simultaneously is stupid.

**Comment by CPTANT** [comment]
I am asking for stamina slowdown to be decouple from health slowdown. Health slowdown won't be affected, that is beyond the scope of the suggestion.

Chainstunning is an attribute of hard stuns, so removing stamina based stuns because of it is dumb.

But yes perhaps reducing stamina after any stun would be a good thing, perhaps the reduction can be directly tied to stun length.

**Comment by Ergovisavi** [comment]
Decoupling it from health slowdown is against the entire point of the object. Removing stamina damage after you've triggered the stun is also against the entire point of the object, the point is to make your health level matter. When you hit 0 health, you go into crit, you have lost the fight, you lose, GG. Similarly, when you are stunned via stamina, you are not intended to get up as though nothing happened and walk off without a care in the world. The problem is that due to disablers being very high in damage due to tasers existing, you hit the "you lose" threshold too quickly.

Hitting 0 health means you lose. If we're going to reset stamina damage after a stun, it makes more sense to simply delete stamina damage altogether and simply work on a stacking status effect.

**Comment by PKPenguin321** [comment]
>the high reward for taser is instant stun. the low reward for disabler is stun after three hits

disabler is low risk due to having high ammo and having the ability to go through windows. once the first hit is landed on somebody, you instantly get a massive reward in the form of them being slowed down, which makes further shots even easier to land and therefore lowers risk even further. upon the second hit, the slowdown, or the reward, is exaggerated even further, and the risk continually lowered. the final hit is simply better than a taser stun by all means. the ONLY case where a disabler is inferior is when the person you're shooting at has a taser and is quick enough to get it out and shoot back before they're ultrafucked by unending slowdown and infinite stuns. in practically every single other case the disabler is ridiculously better, lower risk, and with higher reward. the belief that disablers are worse needs to stop

**Comment by drfino** [comment]
It's a shame we don't have usage stats on taze vs disable mode for sec tasers. I'd put my personal usage and observation of others at 1 in 10 people use taze over disable. If it is so much better, why isn't anyone using it? Defaulting to taze probably has something to do with it, but you're ignoring the other benefits of the taze+drag combo.

Taze is far superior for snatch and grab abductions, and basically any situation where there are more than one threat. Each threat you add triples the amount of shots you need to take.

Disable is very powerful in certain situations, but head to head I think taze is better. 

I think we're off topic though...

**Comment by dannno** [comment]
I don't understand the downvotes on this PR

> If it is so much better, why isn't anyone using it?

What are you talking about? I see disable used all the time since they're far more reliable to hit and with so much more ammo you can just spray down a hallway and then focus your fire when you land a hit and get the slow on your target
